Client/Ecosia/UI/PageAction/PageActionMenu.swift

```diff
@@ -208,11 +211,49 @@ extension PageActionMenu: NotificationThemeable {
 // MARK: Cell
 
 class PageActionMenuCell: UITableViewCell {
+    private weak var badge: UIView?
+    private weak var badgeLabel: UILabel?
+    
     override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
         super.init(style: .subtitle, reuseIdentifier: reuseIdentifier)
     }
 
     required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
+    
+    func isNew(_ isNew: Bool) {
+        if isNew {
+            if badge == nil {
+                let badge = UIView()
+                badge.translatesAutoresizingMaskIntoConstraints = false
+                badge.isUserInteractionEnabled = false
+                badge.layer.cornerRadius = 10
+                contentView.addSubview(badge)
+                
+                let badgeLabel = UILabel()
+                badgeLabel.translatesAutoresizingMaskIntoConstraints = false
+                badgeLabel.font = .systemFont(ofSize: 12, weight: .semibold)
```

Comment:
```suggestion
                badgeLabel.font = .preferredFont(forTextStyle: .footnote).bold()
                badgeLabel.adjustsFontForContentSizeCategory = true
```

for a11y :)

Reply:
I don't think this should use dynamic type, as the badge might grow vertically but the cell won't grow, and will overlap to other cells.

Reply:
if the badge grows, the cell grows with it, won't it? Also the textLabel grows already, so I would strongly advocate to have dynamic type for this.